lib/datasource/docker/readme.md

```diff
@@ -0,0 +1,5 @@
+This datasource fetches the release note from a source repository specified according to the [pre-defined annotation keys of the OCI Image Format Specification](https://github.com/opencontainers/image-spec/blob/main/annotations.md).
+
+Namely, it will extract the value of label `org.opencontainers.image.source` if it exist **on the latest stable tags**.
+
+The [Label Schema](https://label-schema.org/) is now superseded by OCI annotations, and this datasource does not support the `org.label-schema.vcs-url` label.
```

Comment:
```suggestion
The [Label Schema](https://label-schema.org/) is superseded by OCI annotations, therefore this datasource does not support the `org.label-schema.vcs-url` label.
```

We can drop the word `now`.

I think the reason this datasource doesn't support `org.label-schema.vcs-url` is because upstream wants us to use OCI annotations? So I think we should use `therefore` to make relationship between the Label Schema being superseded as the reason that we don't support the Label Schema at all?